src/@types/COLLECT_INFOS.ts

```diff
@@ -3,4 +3,6 @@ enum COLLECT_INFOS {
   COLLECT_DATE = 17,
 }
 
+export const YEARS_WITHOUT_MANUAL_COLLECTION = [2018, 2019, 2020, 2024];
```

Comment:
Eu vejo isso como um problema. Acredito que não é legal uma constante com essa informação, e.g vc vai atualizar isso a cada modificação? Daqui a pouco será outro PR para 2025? Ou se resolvermos coletar algum outro ano?

Particularmente, acredito que seria melhor, a partir da listagem de órgãos, pegar aqueles que não são coletados automaticamente para colocar essa mensagem. Algum mês/ano não tem dados? Coleta manual não realizada. 

"ah mas e se foi feita a coleta e eles que não publicaram os dados?" aí não sei, fica para discussão :face_with_diagonal_mouth:

Reply:
Essa constante seria só mais um "passo manual" da coleta manual, mas acredito que a solução definitiva seria armazenar no banco quais anos fizemos coleta manual e ir atualizando isso dados conforme cada coleta :/

Reply:
Discutido em reunião✅
<em/>Para os anos dos órgãos da coleta manual ficou decidido da seguinte forma: se um órgão for da coleta manual e não tiver dados de algum ano, significa que a coleta manual ainda não foi realizada para esse ano.</em>